tests/operator_test.go

```diff
@@ -1400,6 +1404,9 @@ var _ = Describe("ALL Operator tests", func() {
 			})
 
 			It("should use openshift priority class if not set and available", func() {
+				if utils.IsOpenshift(f.K8sClient) {
```

Comment:
Shouldn't this skip be in the previous test?

Reply:
No, in this test OpenShift already has the `"kubevirt-cluster-critical" PriorityClass`, so I see no reason to use/update it as it won't really test anything. In the previous test we are setting CDI-specific `"system-cluster-critical" PriorityClass`, and test it's passed to the pods, so I think it should not be skipped in OpenShift.